swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
```

Comment:
This looks good! When we have a super simple conditional used to assign a value, we can use a ternary operator as a very concise way to assign said value.

```suggestion
        self.id = id if id is not None else uuid.uuid4().int
```